gamemode/modules/base/sh_createitems.lua

```diff
@@ -461,7 +461,7 @@ function DarkRP.createJob(Name, colorOrTable, model, Description, Weapons, comma
 
     -- Precache model here. Not right before the job change is done
     if istable(CustomTeam.model) then
-        for _, v in pairs(CustomTeam.model) do util.PrecacheModel(v) end
+        for _, v in ipairs(CustomTeam.model) do util.PrecacheModel(v) end
```

Comment:
This one I wouldn't trust: DarkRP keeps this table nice and clean, but servers and/or addons may be doing crazy stuff. Since this is an interface field, I wouldn't trust it to play by the rules.